sentinel-adapter/sentinel-motan-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/motan/BaseMotanSentinelFilter.java

```diff
@@ -0,0 +1,34 @@
+package com.alibaba.csp.sentinel.adapter.motan;
```

Comment:
Could you please prepend the license header here?

Reply:
what lisence, I didn't see it in other files

Reply:
Do you help me to add this?

Reply:
Refer to 
`com.alibaba.csp.sentinel.context.Context`
`com.alibaba.csp.sentinel.adapter.dubbo.SentinelDubboProviderFilter` and other classes.
add lisence header at the top of class file:
```java
/*
 * Copyright 1999-2020 Alibaba Group Holding Ltd.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package xxx;
...
```

Reply:
ok  thank you